src/selectors/authentication.js

```diff
@@ -1,5 +1,27 @@
 import { createSelector } from "reselect";
+import { platformRoles } from "./../constants";
 
 const authData = state => state.get("authentication");
 
 export const selectCurrentUsername = createSelector(authData, data => data.get("name"));
+
+export const selectRolesClaims = createSelector(authData, data => data.get("rolesClaimsValues"));
+
+export const selectGroupRolesClaims = roleGroup =>
+	createSelector(selectRolesClaims, rolesClaims => rolesClaims.get(roleGroup) || null);
+
+export const hasEditorPermissions = roleGroup =>
+	createSelector(selectGroupRolesClaims(roleGroup), appRolesClaims => {
+		if (appRolesClaims != null) {
+			return appRolesClaims.getIn(["*", platformRoles.Editor]);
+		}
+		return false;
+	});
```

Comment:
We need to consider the situation that User can have roles just for specific scope, so we also need to pass current scope here: 
"*"  - it means that Scope=Global - user has rigths for all Scopes
But user can have roles just for some scope:
![image](https://user-images.githubusercontent.com/6649972/107610302-f98b4b00-6c49-11eb-8aa7-c33f94b65c11.png)

Probaly if user does not have Editor permissions for * (Global)  AND for Current Scope - then return FALSE

Reply:
done